.teamcity/Project.kt

```diff
@@ -0,0 +1,97 @@
+import jetbrains.buildServer.configs.kotlin.v2019_2.AbsoluteId
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.CheckoutMode
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.commitStatusPublisher
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.gradle
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+
+
+object Project : Project({
+    buildType(Verify)
+    buildType(ReleasePlugin)
+    params {
+        param("env.GRADLE_ENTERPRISE_ACCESS_KEY", "%ge.gradle.org.access.key%")
+    }
+})
+
+object Verify : BuildType({
+    id = AbsoluteId("VerifyGradleExemplar")
+    uuid = "VerifyGradleExemplar"
+    name = "Verify Gradle Exemplar"
+    description = "Verify Gradle Exemplar"
+
+    vcs {
+        root(AbsoluteId("GradleExemplar_Master"))
+
+        checkoutMode = CheckoutMode.ON_AGENT
+        cleanCheckout = true
+    }
+
+    requirements {
+        contains("teamcity.agent.jvm.os.name", "Linux")
+    }
+
+    triggers {
+        vcs {
+            branchFilter = """
+    +:refs/heads/*
+""".trimIndent()
+        }
+    }
+
+    steps {
+        gradle {
+            useGradleWrapper = true
+            tasks = "check"
+            gradleParams = "--build-cache -Dgradle.cache.remote.username=%gradle.cache.remote.username% -Dgradle.cache.remote.password=%gradle.cache.remote.password%"
```

Comment:
❌ Credentails must not be passed via cmd line arguments. We need to inject this via environment variables instead.

Reply:
done